src/components/NavBar/index.jsx

```diff
@@ -16,7 +16,7 @@ const NavBar = () => {
         <Navbar.Brand>{t("homePage.navbar.home")}</Navbar.Brand>
       </NavLink>
       <Nav className="ml-auto">
-        <NavLink to="/findNgo" className="nav-link">
+        <NavLink to="/findNgos" className="nav-link">
```

Comment:
Is it findNgo or findNow. And shouldn't it be only in Footer, not NavBar?

Reply:
I need to change everything to findNgos, it make more sense than findnow